pkg/sql/conn_executor_exec.go

```diff
@@ -1356,6 +1360,33 @@ func (ex *connExecutor) commitSQLTransactionInternal(ctx context.Context) error
 	return nil
 }
 
+// validateDbZoneConfigs validates zone configs for any modified databases.
+func (ex *connExecutor) validateDbZoneConfigs(ctx context.Context) error {
```

Comment:
nit: looks like this method is not used anymore.

Reply:
Good catch, done.